packages/node/src/authenticatedFetch/fetchFactory.spec.ts

```diff
@@ -42,6 +47,16 @@ const mockNotRedirectedResponse = (): MockedRedirectResponse => {
   };
 };
 
+type MockedStatusResponse = {
```

Comment:
Is this type really required, or the function `mockStatusResponse()` below it? i.e. rather than usage like:
```
fetch.mockResolvedValueOnce(mockStatusResponse(401));
```
...it's actually less code, and easier to read, to just write:
```
fetch.mockResolvedValueOnce({ status: 401 });
```
...but I'm guessing this might be to stop TypeScript complaining? If so, personally I'd still just name this type `MockStatusResponse` (it's only used here twice), and maybe just add a comment to explain why this more convoluted code.

Reply:
No you're right, the intermediary mock function isn't necessary, I anticipated that more properties would need to be passed, and that passing them manually could be cumbersome, but that's not the case. Resolved in bbba942939219f8c7351bee9bfa17ed20ee74dfe